src/app/app.module.ts

```diff
@@ -92,7 +91,6 @@ import { DoctorService } from './services/doctorchat.service';
         BrowserModule,
         BrowserAnimationsModule,
         DndModule.forRoot(),
-        MaterialModule,
```

Comment:
Why is Material Module removed, it works when you install beta 2.0.8 version of Material Module and have the right angular cli version and node version.

Reply:
I thought that since material module has deprecated so we should avoid using the module.